themes/socialbase/socialbase.theme

```diff
@@ -83,10 +84,33 @@ function _socialbase_get_visibility_icon($title) {
 function _socialbase_node_get_comment_count(Node $node, $comment_field_name) {
   $count = 0;
 
-  $comment_count = $node->{$comment_field_name}->comment_count;
-
-  if ($comment_count) {
-    $count = $comment_count;
+  $cids = \Drupal::entityQuery('comment')
```

Comment:
As LU cannot see unpublished comments (or replies on them), but CM+ can I needed a better distinction than we already had.